api/src/main/java/org/openmrs/module/fhirExtension/export/anonymise/impl/AddressResourceRandomiser.java

```diff
@@ -0,0 +1,89 @@
+package org.openmrs.module.fhirExtension.export.anonymise.impl;
+
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.Address;
+import org.hl7.fhir.r4.model.Extension;
+import org.hl7.fhir.r4.model.Patient;
+import org.hl7.fhir.r4.model.StringType;
+import org.openmrs.module.fhirExtension.export.anonymise.ResourceRandomise;
+
+import java.util.List;
+
+public class AddressResourceRandomiser implements ResourceRandomise {
+	
+	private AddressResourceRandomiser() {
+		
+	}
+	
+	public static AddressResourceRandomiser getInstance() {
+		return AddressResourceRandomiser.SingletonHelper.INSTANCE;
+	}
+	
+	@Override
+    public void randomise(IBaseResource iBaseResource, String fixedValue) {
+        Patient patient = (Patient) iBaseResource;
+        if (fixedValue != null && patient.hasAddress()) {
+            List<Address> addressList = patient.getAddress();
+            addressList.forEach(address -> fixedAddressHandler(address, fixedValue));
+            return;
+        }
+        randomHandler(patient);
+    }
+	
+	private void randomHandler(Patient patient) {
+        List<Address> addressList = patient.getAddress();
+        addressList.forEach(this::randomAddressHandler);
+    }
+	
+	private void randomAddressHandler(Address address) {
```

Comment:
randomAddressHandler and fixedAddressHandler runs the same logic(except for fixed value vs random value). Refactor this. Also follow method naming convention